README.md

```diff
@@ -155,6 +154,15 @@ tokenManager: {
 }
 ```
 
+By default, the `expired` event will fire 30 seconds before actual expiration time. You can customize this by setting the `expireEarly` option. If `autoRenew` is set to true, tokens will also be automatically renewed early by this amount.
```

Comment:
Explain why we expire early

Reply:
"..before actual expiration. Renewing tokens slightly early helps ensure a stable user experience. The value should be large enough to account for network latency between the client and Okta's servers."